espnet2/tts/feats_extract/energy.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2020 Nagoya University (Tomoki Hayashi)
```

Comment:
This function might be duplicated in ASR and SE, but it has some unique methods (duration and frame length adjustment), and we can leave it as it is.